tests/Oryx.BuildImage.Tests/Node/NodeJsSampleAppsOtherTests.cs

```diff
@@ -836,7 +836,7 @@ public void BuildsApp_ByRunningNpmInstall_AndCustomBuildCommand()
             var subDir = Guid.NewGuid();
             var script = new ShellScriptBuilder()
                 .SetEnvironmentVariable(
-                SettingsKeys.CustomNpmRunBuildCommand,
+                SettingsKeys.CustomRunBuildCommand,
                 $"echo > /tmp/foo.txt")
```

Comment:
is this the only buildimage test where we are testing this feature?

Reply:
yes, `NodePlatformTest` unit test has good amount of tests for this so I added only test for this. Given how long it takes our builds to run, I feel we should try to write as many unit tests as possible which cover the scenarios. This way tests run fast and we can add lot more variations there and add few integration tests.